deploy/cephfs/kubernetes/cephfs-storage-class.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-cephfs
+provisioner: csi-cephfsplugin
+parameters:
+  # The driver can use either ceph-fuse (fuse) or ceph kernel client (kernel)
+  mounter: fuse
```

Comment:
put the storage class on par with existing cephfs storage class, don't set mounter here.

Reply:
if not here, what would be the correct place to store such configuration?

Reply:
probe cephfs-fuse first, if not exit, fall back to kernel cephfs 

see https://github.com/kubernetes/kubernetes/pull/55866

Reply:
But it can be assumed that both fuse and kernel client should be present anyway, since that's what the cephfsplugin Docker image contains, no? And one of our use-cases would be to override the defaults and use the kernel client instead, for performance reasons.

Reply:
the probe idea tries one at a time till mounted. In this case, you can either make mounter order hard coded or based on a cli option. You can first try cephfs-fuse, if works then done, if not, try kernel cephfs.

Reply:
It's fine to have a probe if not given, but we would like our users to be able to choose between using ceph-fuse or the kernel client explicitly. The best i think is to let them pass it (optionally) in the storage class.